README.md

```diff
@@ -320,6 +320,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 
 ## Implementations/Compilers
  * [charly](https://github.com/charly-lang) - Charly Programming Language
+ * [cibyl](https://github.com/senselogic/CIBYL) - Lightweight language which compiles to Ruby and Crystal.
```

Comment:
Avoid a full stop (dot at the end) for consistency with other entries.

Reply:
Oops :$